backend/tprm/views.py

```
@@ -1,10 +1,15 @@
 from rest_framework.response import Response
-from iam.models import Folder
+from iam.models import Folder, RoleAssignment, UserGroup
```

Ruff (0.8.2)

`iam.models.UserGroup` imported but unused

Remove unused import: `iam.models.UserGroup`

(F401)